### PR TITLE
Chapter 8 - Scrum Master Anti-Patterns

### PR DESCRIPTION
**Pull request by SaarthakDeshmukh**
Completed chapter 8 anti-patterns of SM

### CODE REVIEW

**Review by udeysingh95 — APPROVED** [review]
What do you think is the most common anti-pattern? and what do you think is the most harmful anti-pattern and why?

**Comment by SaarthakDeshmukh** [comment]
@udeysingh95 I think the most common anti-pattern of SM is that they assign tasks to the team members and not let them self-organize, and most harmful would be that they solve the problems of the team and not let them solve and another one would be they allow the PO on committing to the oversized backlog.

File: 7-scrum-master-role/8-anti-patterns.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+**3. No Retro:** The Scrum Master does not gather data during the Sprint that supports the team in the upcoming retrospective. This could also be a sign of frustration.\
+**4. No Documentation:** The Scrum Master does not take minutes for later use. A retrospective is a substantial investment, and the Scrum team should take it seriously. Taking notes and photos supports this process.\
+**5. No psychological safety:** The retrospective is an endless cycle of blame and finger pointing without intervention from the Scrum Master. The team wins together, and the team loses together. The blame game documents both the failure of the Scrum Master as the facilitator of the retrospective as well as the team’s lack of maturity and communication skills.\
+**6. Bullying is accepted:** One or two team members are dominating the retrospective. This communication behavior is often a sign of either a weak or uninterested Scrum Master.\
+**7. Stakeholder alert:** The Scrum Master permits stakeholders to participate in the retrospective. The retrospective is off-limits to stakeholders, and the Scrum Master needs to enforce this rule.
+
+### Scrum Master Antipatterns that May Impair Your Team
```
Comment (Vaibhavpratapsingh22) [inline 7-scrum-master-role/8-anti-patterns.md:77]:
```suggestion
### Scrum Master Anti-Patterns that May Impair Your Team
```

File: 7-scrum-master-role/README.md [hunk 2]
```diff
@@ -25,3 +25,7 @@ This chapter deals with one of the many stances of Scrum Master which is Teachin
 ## Chapter 7 - Coaching
 
 In this chapter we will talk about a very important part of Scrum Master's journey which is Coaching, we will see what is coaching, what is not, who does SM coach, and What. Few other qualities which a coach should possess like, Actively Listening and asking powerful questions.
+
+## Chapter 8 - Scrum Master Anti-Patterns
```
Comment (Vaibhavpratapsingh22) [inline 7-scrum-master-role/README.md:29]: Add some more content

File: 7-scrum-master-role/8-anti-patterns.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+<p align="center"> <img width="500" height="" src="https://age-of-product.com/wp-content/uploads/28-capacity-sprint-planning-slack-time-overutilization-1650.jpg"></p>
+
+### Scrum Master Anti-Patterns During the Sprint
+
+The Sprint is an event in itself that contains all the work and all the other events that happen during the time-boxed period of development. The following anti-patterns focus on the mishandling of the Sprint itself:
+
+**1. Flow disruption:** The SM allows stakeholders to disrupt the flow of the Scrum team during the Sprint. There are several possibilities of how stakeholders can interrupt the flow of the team during a Sprint. Any of the examples will impede the team’s productivity. The SM must prevent them from manifesting themselves:
+
+- The SM has a laissez-faire policy as far as access to the development team is concerned.
```
Comment (Vaibhavpratapsingh22) [inline 7-scrum-master-role/8-anti-patterns.md:53]: Bullet points never end with a full stop.

**Review by Vaibhavpratapsingh22 — CHANGES_REQUESTED** [review]
## Excellent Work 🍰 
Some small changes from my side else the content is written very nicely and I understood all the topics discussed here.

File: 7-scrum-master-role/8-anti-patterns.md (hunk 3 above)
Comment (udeysingh95) [reply 7-scrum-master-role/8-anti-patterns.md:53]: They do, they do!

Comment (SaarthakDeshmukh) [reply 7-scrum-master-role/8-anti-patterns.md:53]: It's a tricky thing some times they are needed and some times they are not. It all depends on the context. @Vaibhavpratapsingh22 It's my bad that I wrote this on your previous PR. I should have checked it. Sorry about that!

**Comment by udeysingh95** [comment]
I would like to hear one more answer on the same question. 👍

**Review by Vaibhavpratapsingh22 — APPROVED** [review]

**Comment by SaarthakDeshmukh** [comment]
One more very common anti-pattern is an enforcer, he should not enforce anything on the team, rather let the team decide if they find it helpful of not, he's duty is to teach them the right practices at the right time, but it's the team's duty to select. 
Most Harmful would be guarding the team by negative comments from let's say Stakeholders, which results in no progress. Plus also many times it happens that the team is not willing to have a retro at the end and SM allows that to happen, this shouldn't happen, he should rather explain the importance of retro and get the team on the same page. @udeysingh95 What are your views?

**Pull request merged**